### PR TITLE
react2-week1

### PR DESCRIPTION
**Pull request by Hanouj**

### CODE REVIEW

**Review by Hailegebriel22 — COMMENTED** [review]
@Hanouj  You did a great job. I leave some comments hoping that they would be helpful.

File: react/react2/week1/react2-week1/src/SearchUser.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    setIsLoading(true);
+    setError(undefined);
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        if (data && data.items) {
+          setUsers(data.items);
+          console.log(data.items);
+        } else {
+          setError("Error Slow down!");
```
[the remark this answers is not in the document]
Comment (Hanouj) [reply react/react2/week1/react2-week1/src/SearchUser.jsx:29]: I thought about that, how can I use its own response error ?

File: react/react2/week1/react2-week1/src/SearchUser.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    const API_URL = `https://api.github.com/search/users?q=${query}`;
+    console.log(query);
+    setIsLoading(true);
+    setError(undefined);
+    fetch(API_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        if (data && data.items) {
+          setUsers(data.items);
+          console.log(data.items);
```
[the remark this answers is not in the document]
Comment (Hanouj) [reply react/react2/week1/react2-week1/src/SearchUser.jsx:27]: I will remove it ☺️ thanks

File: react/react2/week1/react2-week1/src/RenderUser.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+export const RenderUser = () =>{
+    const {
+        users,
+        isLoading,
+        error     
+    }=useContext(UserContext)
+
+     const usersResult = users.map((user, index) => (
+        <div key={index}>
```
[the remark this answers is not in the document]
Comment (Hanouj) [reply react/react2/week1/react2-week1/src/RenderUser.jsx:13]: Yes you right

File: react/react2/week1/react2-week1/src/SearchUser.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (annamourasobral) [reply react/react2/week1/react2-week1/src/SearchUser.jsx:29]: Thank you, @Hailegebriel22 for the input! 
You can use the response error of try/catch, @Hanouj 
This gives you a more specific kind of error.

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Good job, @Hanouj, you are a star! 
You can just implement @Hailegebriel22 ' suggestion and go forward to merge.
Happy coding! 👩🏻‍💻

**Pull request merged**